l10n_it_fiscalcode/model/res_partner.py

```diff
@@ -1,31 +1,55 @@
 # -*- coding: utf-8 -*-
 # Copyright 2014 Associazione Odoo Italia (<http://www.odoo-italia.org>)
 # Copyright 2016 Andrea Gallina (Apulia Software)
+# Copyright 2016 Giuliano Lotta
+# version 0.1- changed method check_fiscalcode() to consider multiple situation
+#            - removed local "individual" field, to use "is_company" original res.partner.is_compamy field
+#            - prefixed (_) constrain method, to adhere with OCA convention
+#            - https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#oca-guidelines
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+# 1: imports of python lib
+# 2: import of known third party lib
+# 3:  imports of openerp
 from odoo import models, fields, api
 
 
 class ResPartner(models.Model):
+    # Private attributes
     _inherit = 'res.partner'
 
+    # Fields declaration
+    fiscalcode = fields.Char(
+        'Fiscal Code', size=16, help="Italian Fiscal Code")
+
+    # Constraints and onchanges
     @api.multi
-    def check_fiscalcode(self):
+    def _check_fiscalcode(self):
```

Comment:
ciao Fabrizio,
la funzione è un contrain (ora in api.constrain) richiamato internamento nella fase di scrittura record. 
Non viene invocato "esternamente". Modifica sicura.